rules/opentitan/sim_dv.bzl

```diff
@@ -148,6 +148,12 @@ def _test_dispatch(ctx, exec_env, provider):
 
         # Collect the DV logs databases if they're provided.
         data_files.extend(getattr(provider, "logs", []))
+
+        # Collect the elf output file if provided.
+        elf_file = getattr(provider, "elf", None)
+        if elf_file:
```

Comment:
IIUC, shouldn't all [SimDv ExecEnvs provide an elf file](https://cs.opensource.google/opentitan/opentitan/+/master:rules/opentitan/sim_dv.bzl;drc=6e6d206405bae9a932fc8dda97ad5bd74498a274;l=91)? should we error out here if not?

Reply:
I think it would make sense to error out, @cfrantz what are your thoughts?

I will submit this change to continue to make progress with other tests, but will submit a follow up change once we are ready to close this comment thread.